cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -482,23 +482,16 @@ public boolean hasOngoingExecution() {
     private Set<Integer> _recentlyRemovedBrokers;
     private final long _executionStartMs;
     private Throwable _executionException;
-    private final UserTaskManager.UserTaskInfo _userTaskInfo;
+    private UserTaskManager.UserTaskInfo _userTaskInfo;
```

Comment:
Looks like `_userTaskInfo` is used only in `ProposalExecutionRunnable#execute()` -- can we make it a local variable in this function rather than making it a field of the class?